options/locale/locale_en-US.ini

```
@@ -1625,6 +1625,8 @@ pulls.reopened_at = `reopened this pull request <a id="%[1]s" href="#%[1]s">%[2]
 pulls.merge_instruction_hint = `You can also view <a class="show-instruction">command line instructions</a>.`
 pulls.merge_instruction_step1_desc = From your project repository, check out a new branch and test the changes.
 pulls.merge_instruction_step2_desc = Merge the changes and update on Gitea.
+pulls.clear_merge_message = Clear merge message
+pulls.clear_merge_message_hint = Clearing the merge message will only remove the commit message content and keep generated git trailers.
```

> will keep generated git trailers

I can imagine that this will cause a lot of confusion as no one knows what `git trailers` are supposed to be.
I'm assuming you mean something like `Reviewed-By`, `Co-Authored-By`, …?
Perhaps we should clarify this here with an example as well.

---

Where should this be explained? Or do you think it would be enough to put an example in parantheses?

---

I meant exactly something like an example in parentheses behind.
I simply wasn't sure if you meant what I think you mean with `git trailers`.
So, for example:
```suggestion
pulls.clear_merge_message_hint = Clearing the merge message will only remove the commit message content and keep generated git trailers (i.e. "Co-Authored-By …").
```

---

Makes sense! I'll apply that suggestion.